lib/path.c

```diff
@@ -64,23 +68,37 @@ path_vfopen(const char *mode, int exit_on_error, const char *path, va_list ap)
 {
 	FILE *f;
 	const char *p = path_vcreate(path, ap);
-
+	if (!p) {
+		p = "vpath";
```

Comment:
It would be better to use 
```
if (!p)
   goto err;
...
err:
   if (exit_on_error) {
     if (p)
         err(EXIT_FAILURE, _("cannot open %s"), p);
    else
        err(EXIT_FAILURE, _("path no specified"));